rdl/shared/Providers.py

```diff
@@ -0,0 +1,25 @@
+class __AuditColumnsNames:
+    def __init__(self):
+        self.audit_column_prefix = 'rdl_'
```

Comment:
This means the consumer has to set the prefix as `prefix_` rather than `prefix`.. shouldn't we automatically be including the `_` in the properties down below?

Reply:
I agree that it is a bit of a smell but I can think of a couple of reasons not to do this:

* people may want no prefix (with means we will have a leading underscore)
* people may want to use a different separator from the column names